### PR TITLE
CS: various minor code changes

### PR DESCRIPTION
**Pull request by jrfnl**
### CS: remove PHP close tags

As discussed in #3:

> The `?>` at the end of a file is optional and nowadays,
> the consensus is to leave it out as it will prevent
> typical issues with "headers already send" due to
> the whitespace at the end of the file being interpreted
> as HTML whitespace and send to the server.

Refs:
* https://www.php.net/manual/en/language.basic-syntax.phptags.php
* https://www.php-fig.org/psr/psr-12/#22-files

> If a file contains only PHP code, it is preferable to
> omit the PHP closing tag at the end of the file.
> This prevents accidental whitespace or new lines being
> added after the PHP closing tag, which may cause unwanted
> effects because PHP will start output buffering when
> there is no intention from the programmer to send
> any output at that point in the script.

_(From the PHP manual)_

### CS: use curly braces for all control structures

Control structures without curly braces are quite
error prone as they will only apply to the
_first_ PHP statement after the condition.
This means that a second statement - even when
indented the same - will be run _unconditionally_,
but it also means that anyone working with the code
needs a really good understanding of what
a "statement" is in PHP.

In other words, it's best practice by now
to always use curly braces with control structures.

Ref:
* PSR12: https://www.php-fig.org/psr/psr-12/#5-control-structures
    > The body of each structure MUST be enclosed by
    > braces. This standardizes how the structures
    > look and reduces the likelihood of introducing
    > errors as new lines get added to the body.

### CODE REVIEW

**Review by solardiz — APPROVED** [review]
Looks good to me.

In the below paragraph, maybe correct "send" to "sent" in two places:

```
> The `?>` at the end of a file is optional and nowadays,
> the consensus is to leave it out as it will prevent
> typical issues with "headers already send" due to
> the whitespace at the end of the file being interpreted
> as HTML whitespace and send to the server.
```

**Comment by jrfnl** [comment]
Done.
